packages/core/echo/echo-signals/package.json

```diff
@@ -37,6 +37,7 @@
       ]
     }
   },
+  "sideEffects": true,
```

Comment:
what's this do?

Reply:
not relevant now, but it signals bundlers that a bare import has side effects:

```
// wont be removed during tree shaking
import '@dxos/echo-signals/react';
```